internal/ime_generate/manifest_generate.go

```diff
@@ -0,0 +1,63 @@
+package ime_generate
+
+import (
+	"io/ioutil"
+	"reflect"
+	"strconv"
+
+	"github.com/Agaetis-IT/imerologio-cli/pkg/ime_types"
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/cheggaaa/pb/v3"
+)
+
+type manifestReader struct {
+	answers        *ime_types.Answers
+	strimziVersion string
+}
+
+func dumpQuestionParameters(answers ime_types.Answers, questions []*survey.Question, result *string) {
+	value := reflect.ValueOf(answers)
+
+	for _, q := range questions {
+		field := value.FieldByName(q.Name)
+		switch field.Type().String() {
+		case "bool":
+			*result += q.Name + ": " + strconv.FormatBool(field.Bool()) + "\n"
+		default:
+			*result += q.Name + ": " + field.String() + "\n"
+		}
```

Comment:
As discuss, we should generate an idempotent manifest. The `Name` seems weak to this because it's subject of renaming/refactoring. Maybe we can add a unique id to each field ?